simple_history/tests/models.py

```diff
@@ -432,3 +432,28 @@ class UUIDDefaultModel(models.Model):
 
 # Clear the SIMPLE_HISTORY_HISTORY_ID_USE_UUID
 delattr(settings, 'SIMPLE_HISTORY_HISTORY_ID_USE_UUID')
+
+
+# Set the SIMPLE_HISTORY_HISTORY_CHANGE_REASON_FIELD and MAX_LENGTH
+setattr(settings, 'SIMPLE_HISTORY_HISTORY_CHANGE_REASON_FIELD', models.TextField)
```

Comment:
Here's the line. You don't instantiate the `TextField` here. And you don't instantiate it when you call the setting either. That's why the field isn't expected.

Reply:
Oh, makes sense. Thanks!